zstacklib/zstacklib/gpu/vendors/kunlunxin.py

```
@@ -0,0 +1,324 @@
+            elif current_section == "Utilization":
+                if "Xpu" in line and "%" in line:
+                    util_str = line.split(":")[1].strip().rstrip("%")
+                    utilization = cls.parse_unit_value(util_str)
+            elif "Power Draw" in line:
+                power_str = line.split(":")[1].strip().rstrip("W")
+                power_draw = cls.parse_unit_value(power_str)
+            elif "XPU Current Temp" in line:
+                temp_str = line.split(":")[1].strip().rstrip("C")
+                temperature = cls.parse_unit_value(temp_str)
+
+        # 统一计算内存利用率，避免对输出顺序的依赖
```

Ruff (0.14.13)

Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)